torch/_inductor/ir.py

```diff
@@ -7044,14 +7266,40 @@ def create_out_of_place(
         cpp_kernel_name = kernel._name
         python_kernel_name = cpp_kernel_name.replace("::", ".")
         with V.graph.fake_mode:
-            (
-                example_output,
-                tensor_args,
-                non_tensor_args,
-                unflatten_args,
-                unbacked_bindings,
-            ) = cls.process_kernel(kernel, inputs, *args, **kwargs)
-        assert not unbacked_bindings, f"{kernel}, {unbacked_bindings}"
+            if isinstance(inputs, MultiOutput) and isinstance(
```

Comment:
the first two cases look the same, can we combine?
After merging all `process_kernel` into one, you can use a single flag as input arg to remove branching here.